### PR TITLE
Validate CROWN

### PR DESCRIPTION
**Pull request by OrmEmbaar**
# Validate [CROWN](https://solscan.io/token/GDfnEsia2WLAW5t8yx2X5j2mkfA74i5kwGdDuZHt7XmG)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/photofinishgame/status/1688286122311401472
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/crown-by-third-time-games

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Review by DavidChapman77 — COMMENTED** [review]
Validating that the info is correct.

> * [x]   Mint address: GDfnEsia2WLAW5t8yx2X5j2mkfA74i5kwGdDuZHt7XmG
> * [x]   Symbol: CROWN
> * [x]   Token Decimals: 9
> * [x]   Token icon: https://shdw-drive.genesysgo.net/AwJ6W2rRaYCGXimceFuLm5td14fhN1VFEfSYg566RxMD/image.png
> * [x]   On-chain metadata matches
> * [x]   No duplication on Jupiter's strict list

**Comment by samuelvanderwaal** [comment]
I am a former Metaplex Protocol developer where I was a core maintainer of the Token Metadata program and am also the author of a popular [CLI tool](https://github.com/samuelvanderwaal/metaboss) for NFTs on Solana. My [Twitter](https://twitter.com/samvwaal).

I have validated the following:

- [x]  Mint address: GDfnEsia2WLAW5t8yx2X5j2mkfA74i5kwGdDuZHt7XmG
- [x]  Symbol: CROWN
- [x]  Token Decimals: 9
- [x]  Token icon: https://shdw-drive.genesysgo.net/AwJ6W2rRaYCGXimceFuLm5td14fhN1VFEfSYg566RxMD/image.png
- [x]  On-chain metadata matches
- [x]  No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -753,4 +753,5 @@ JEFF,JEFF,FkCaFsprX7gySagFoQPHNbe9MRkjrvh21cokJo6C1e2T,6,https://arweave.net/Pi-
 DIG Gold,DIG,HQMBNWptHTTkGY5FDAypqNqQzx9jF6cNJEZyrxoMMAVs,9,https://shdw-drive.genesysgo.net/6Y4jCS21YEVcEiL5qfifYRk6gQ1ozkWs8LsmSTAJeQR6/SPL_TOKEN.png,true
 ASTRALIS,ASTRALIS,ASTRALvKjGK2xk2pamjMBU5dav5cEQa6zpKCP6FZ7BAJ,6,https://shdw-drive.genesysgo.net/AcG1tjtJ3vR1XzSCcVArheGLrwmMsphqExa6VodLymGF/astralis_1500x1500_8bit.png,true
 GEM,GEM,E7BGDtpNXUTqPNbZxKHiLowgLddiAeuKcByD7tSnfYWD,5,https://bafkreifpm7rbwohp4yys2guyibfzqbctu3bist6zetmyq456om2rqvk544.ipfs.nftstorage.link/,true
-SOUL,SOUL,F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf,9,https://arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ,true
\ No newline at end of file
+SOUL,SOUL,F6weWmuc1vwdL4u38Ro9jKXHEMjP9BoNdWMF5o5TvtJf,9,https://arweave.net/ipj9IdEAQAXt0LS6lOt7jQY7W4iazsVFBkN60U7M4iQ,true
+CROWN,CROWN,GDfnEsia2WLAW5t8yx2X5j2mkfA74i5kwGdDuZHt7XmG,9,https://shdw-drive.genesysgo.net/AwJ6W2rRaYCGXimceFuLm5td14fhN1VFEfSYg566RxMD/image.png,true
```
Comment (Timoon21) [inline validated-tokens.csv:757]:
```suggestion

CROWN Token,CROWN,GDfnEsia2WLAW5t8yx2X5j2mkfA74i5kwGdDuZHt7XmG,9,https://shdw-drive.genesysgo.net/AwJ6W2rRaYCGXimceFuLm5td14fhN1VFEfSYg566RxMD/image.png,true
```

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

**Pull request merged**